pkg/cloudprovider/providers/photon/photon.go

```diff
@@ -512,6 +517,7 @@ func (pc *PCCloud) Zones() (cloudprovider.Zones, bool) {
 	return pc, true
 }
 
+// GetZone implements Zones.GetZone, Zones are currently not supported.
```

Comment:
@saravanan30erd Maybe I'm missing something here, but this function returns the zone, right? `GetZoneByProviderID` and `GetZoneByNodeName` are the ones which haven't been implemented yet.

If this makes sense, can you update the comment to https://github.com/kubernetes/kubernetes/pull/68399#discussion_r228750097?

Reply:
@nikhita I read the initial comments in that file and put the comment, but I should have checked more.

Reply:
> @nikhita I read the initial comments in that file and put the comment, but I should have checked more.

No problem! :)